web/scripts/views/paysuccess.hbs

```diff
@@ -0,0 +1,3 @@
+<html>
+  <p> Your payment is successful!!! </p>
+  </html>
```

Comment:
You can delete the views folder (paysuccess and index). I won't be using that code for integrating this into the shopping cart.